helm/qontract-reconcile/templates/template.yaml

```diff
@@ -773,6 +902,14 @@ parameters:
   value: ''
 - name: PUSHGATEWAY_SECRET_NAME
   value: push-gateway-auth-url
+### APPSRE-8096: Vector Section Begins
+- name: VECTOR_IMAGE
+  value: timberio/vector
+- name: VECTOR_IMAGE_TAG
+  value: 0.38.0-debian
+- name: VECTOR_IMAGE_PULL_POLICY
+  value: Always
```

Comment:
Do we use `IfNotPresent` elsewhere?

Reply:
I'll make this change as well.

Reply:
@TGPSKI I had a look again while I was going to change this (with `sed`) and we use `Always` for pull policies in many places. I'm sort of inclined to leave this as `Always`. Do you feel strongly to change this one to `IfNotPresent`?

Reply:
It's just about saving data transfer costs. To me, no value in always pulling if we have a cached image available locally. 

What is the value in always pulling?

Reply:
The value is in maintianing consistency across all instances in this config. They should either all be `IfNotPresent` or none. I have no opinion on which is chosen.